net-news/snownews/snownews-1.9.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit toolchain-funcs
+
+DESCRIPTION="Command-line RSS feed reader"
+HOMEPAGE="https://github.com/msharov/snownews"
+SRC_URI="https://github.com/msharov/snownews/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE=""
+
+COMMON_DEPEND="
+	>=dev-libs/libxml2-2.5.6
+	sys-libs/ncurses:=[unicode(+)]
+	dev-libs/openssl
+	net-misc/curl
+	sys-devel/gettext
+"
+RDEPEND="
+	${COMMON_DEPEND}
```

Comment:
I don't see the logic here. Instead adding ```${COMMON_DEPEND}```, you could also add them do ```DEPEND``` and include in ```RDEPEND```?

Reply:
That's just how the 1.6 ebuild did things, but yeah, now that you've pointed it out it makes much more sense.